armi/reactor/converters/tests/test_axialExpansionChanger.py

```diff
@@ -351,14 +350,14 @@ def complexConservationTest(self, a):
         newMasses, newNDens = self._getComponentMassAndNDens(a)
         # make sure that the assembly returned to the original state
         for orig, new in zip(origMesh, a.getAxialMesh()):
-            self.assertAlmostEqual(orig, new, places=12, msg=f"{a}")
+            self.assertAlmostEqual(orig, new, places=8, msg=f"{a}")
```

Comment:
Recovering block heights is a key part of recovering mass. I would prefer to keep this check tight as well.